.env.example

```diff
@@ -0,0 +1,7 @@
+ETH_ACCOUNT_MNENOMIC="a dozen random words"
+INFURA_API_KEY=<API_KEY>
```

Comment:
Where does this come from? Could you add a paragraph or two to the dev handbook?